Gemfile.lock

```diff
@@ -315,76 +316,78 @@ GEM
       momentjs-rails (>= 2.8.1)
     brakeman (5.4.1)
     builder (3.2.4)
-    bundler-audit (0.9.0.1)
+    bundler-audit (0.9.1)
       bundler (>= 1.2.0, < 3)
       thor (~> 1.0)
     byebug (8.2.5)
-    chef-utils (18.1.29)
+    chef-utils (18.4.2)
```

Comment:
### [chef-utils](https://github.com/chef/chef/tree/main/chef-utils)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 18.1.29 → 18.4.2

[_[change-log](https://github.com/chef/chef/blob/main/CHANGELOG.md), [source-code](https://github.com/chef/chef/tree/main/chef-utils), [gem-diff](https://my.diffend.io/gems/chef-utils/18.1.29/18.4.2)_]





<details>
<summary>Commits</summary>

A change of **403** commits. See the full changes on [the compare page](https://github.com/chef/chef/compare/v18.1.29...v18.4.2).

These are the first 10 commits:
- (448d438) [Bump version to 18.2.31 by Chef Expeditor](https://github.com/chef/chef/commit/448d438e645b00677b4c2197d1446bc508f9baec)
- (b3cbd76) [Update for chef-foundation ver to pick openssl-1.0.2ze](https://github.com/chef/chef/commit/b3cbd76ac1609dd9a488aefc4d4e747154f0c45d)
- (8576ef3) [remove openssl-src](https://github.com/chef/chef/commit/8576ef3d33562e80991b6c45329cb338d79685a5)
- (72fd888) [Adding double-quotes around path for Windows 'secedit' comma…](https://github.com/chef/chef/commit/72fd88829b3ca16a49ef39af22dcc0bc6c64e10d)
- (c89c12b) [Make action homebrew_bin_path public which is used in other …](https://github.com/chef/chef/commit/c89c12b0d14978f9957c6f39b50ef843eea94ed9)
- (1e7a3bb) [Bump version to 18.2.32 by Chef Expeditor](https://github.com/chef/chef/commit/1e7a3bb5728963b135f7699356def6e8fcce00fe)
- (01c23ea) [Optimize Chef::CookbookSynchronizer#remove_deleted_files (#1…](https://github.com/chef/chef/commit/01c23ea73487dc9b6ee86edc7b797c12e3b1a14a)
- (3d7c78e) [Bump version to 18.2.33 by Chef Expeditor](https://github.com/chef/chef/commit/3d7c78e8afa1ef54e8ded8e914daf727607b3fb3)
- (5ab62ef) [update ::File.exists? to ::File.exist?](https://github.com/chef/chef/commit/5ab62efe29f632dd5d5f22f90db0f352f43cdf94)
- (be390a7) [Merge pull request #13807 from chef/poorndm/test-openssl-1.0…](https://github.com/chef/chef/commit/be390a72a055375bca4ea1861c57a1ce99da0710)


</details>